packages/codemods/src/modRunner/modFilter.ts

```diff
@@ -0,0 +1,33 @@
+import { CodeMod } from '../codeMods/types';
+import { Maybe } from '../maybe';
+
+export function getStringFilter(filter: string) {
+  const filterString = filter;
+  return (modString: string) => {
+    return modString === filterString;
+  };
+}
```

Comment:
Can be shortened to
`export const getStringFilter = (f: string) => (modString: string) => f === modString;`

Is that better? Maybe not, definitely denser though.